content/applications/services/helpdesk/overview/sla.rst

```diff
@@ -150,19 +157,19 @@ the chart.
 
        .. image:: sla/sla-report-bar.png
           :align: center
-          :alt: View of the SLA status analysis report in bar view
+          :alt: View of the SLA status analysis report in bar view.
```

Comment:
I can't make a comment in line, but for line 150, I would say 'SLA Status Analysis` report since SLA is in the name in the menu when you click on it,

Reply:
Also, in line 152, I would mention the buttons to change the view are in the top right of the chart, then they appear above the chart. I couldn't not find them at first and didn't realize they were tucked all the way in the corner.